xml/release-notes.xml

```diff
@@ -398,6 +410,87 @@
       <package>kdesdk-scripts</package>.
      </para>
     </listitem>
+    <listitem>
+     <para>
+      <package>kdeuser</package>: Replaced by
+      <package>kde-user-manager</package>.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      <package>kile5</package>: Replaced by
+      <package>kile</package>.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      <package>kpassgen</package>
+      <package>kpassx</package>: Both replaced by
+      <package>kpassxc</package>.
```

Comment:
keepassx and keepassxc

Reply:
This doesn't seem like a straightforward replacement tbh -- kpassgen is a password generator, whereas keepassx(c) is a full password manager. Maybe instead of "x replaced by y" write something like "Use a different password generator, such as the one included with keepassxc".

Reply:
Well, keepassxc allows generating password without storing them. It replaces kpassgen and offers more features